javascript/javascript1/week3/note-pad.js

```
@@ -0,0 +1,43 @@
+const notes = [];
+
+function saveNote(content, id) {
+    //Code goes here here
+    notes.push({ content: content, id: id });
+}
+saveNote("Attend classes regularly", 1);
+saveNote("Pay attention to lessons", 2);
+saveNote("Ask for doubts", 3);
+saveNote("Do your homework", 4);
+
+console.log("\n saveNote function outputs \n");
+console.log(notes);
+
+// 2'nd function
+function getNote(id) {
+    for (let i = 0; i < notes.length; i++) {
+        if (notes[i].id === id) {
+            return notes[i];
+        }
+    }
+    return "Error!";
```

instead of return string, you can throw an error
```js
throw new Error('note not found!');
```